pinot-broker/src/test/java/org/apache/pinot/broker/queryquota/HelixExternalViewBasedQueryQuotaManagerTest.java

```diff
@@ -18,6 +18,8 @@
  */
 package org.apache.pinot.broker.queryquota;
 
+import com.google.common.util.concurrent.RateLimiter;
+import com.google.common.util.concurrent.TestRateLimiter;
```

Comment:
I don't find this class in guava source code (I've looked for it in https://github.com/google/guava/blob/v33.3.1 and master).

Reply:
TestRateLimiter is part of the PR. It's in src/test/com.google* package to work around the visibility issues.

Reply:
Wow! That was unexpected. Unless a major force, never, ever, ever create a class on a third party package. That is basically forbidden when using module paths. If there is a reason why we need to do it here (like being able to call a package protected method) it is a very strong reason to do not use RateLimiter API.

Reply:
Ok, I'll move the change if it's so problematic. Out of curiosity - what damage this class could do being limited to tests that explicitly set it up and only used in a single test ?